get_s3.go

```diff
@@ -127,17 +127,17 @@ func (g *S3Getter) Get(dst string, u *url.URL) error {
 			req.Marker = aws.String(lastMarker)
 		}
 
-		resp, err := client.ListObjectsWithContext(ctx, req)
+		resp, err := client.ListObjects(ctx, req)
 		if err != nil {
 			return err
 		}
 
-		hasMore = aws.BoolValue(resp.IsTruncated)
+		hasMore = resp.IsTruncated
 
 		// Get each object storing each file relative to the destination path
 		for _, object := range resp.Contents {
-			lastMarker = aws.StringValue(object.Key)
-			objPath := aws.StringValue(object.Key)
+			lastMarker = *object.Key
```

Comment:
```suggestion
			lastMarker = aws.ToString(object.Key)
```
because this might be nil